components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/utils/AuthUtil.java

```diff
@@ -168,7 +174,10 @@ public static NewCookie cookieBuilder(String name, String value, String path, bo
         if (expiresIn != null && !expiresIn.isEmpty()) {
             stringBuilder.append(COOKIE_VALUE_SEPERATOR).append(expiresIn);
         }
-        return new NewCookie(name, stringBuilder.toString());
+
+        //Append unique environment name in deployment.yaml
+        String environmentName = ConfigurationService.getEnviornmentName();
```

Comment:
Same mistake here also in method name "getEnviornmentName()"

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/4702/commits/7c3c6206a6826c02f027cc5c99e7cfdb4608d36d